FAQ.md

```diff
@@ -0,0 +1,17 @@
+# Questions and answers
+
+The goal of this file is to have a place to easily commit answers to questions
+in a way that's easily searchable, and can make its way into official
+documentation later.
+
+## Q: How do I provide static IP addresses?
+
+As of OpenShift 4.2, by default the kernel command line arguments
+are persisted.  See this PR: https://github.com/coreos/ignition-dracut/pull/89
+
+In both cases, you can use the traditional Red Hat Linux `/etc/sysconfig/network-scripts`
```

Comment:
This feels a bit confusing. Maybe instead of "in both cases", we can elaborate a bit by saying: you can set static IP either by:
 - setting it via the kernel command line and having it persist
 - dropping in an ignition snippet to create a `/etc/sysconfig/network-scripts`

Reply:
Mmm...not quite, because if you just do the latter, the first boot is DHCP.  I reworded the text.